warehouse/internal/api/http.go

```diff
@@ -146,18 +152,14 @@ func (api *WarehouseAPI) processHandler(w http.ResponseWriter, r *http.Request)
 		return
 	}
 
-	api.Stats.NewTaggedStat("rows_staged", stats.CountType, stats.Tags{
```

Comment:
We don't need `warehouseID`. Also, we are only passing ID for Source and Destination and nothing else, so `Destination.DestinationDefinition.Name` will come as empty. Removing it.